introduction.md

```diff
@@ -1,2 +1,12 @@
 Hi everyone! My name is jnguyen7410 and I'll be doing the Mastering Github course!
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?